Content.Server/Power/Components/ChargerActiveComponent.cs

```diff
@@ -0,0 +1,10 @@
+using Content.Shared.Containers.ItemSlots;
+using Content.Shared.Power;
+
+namespace Content.Server.Power.Components
+{
+    [RegisterComponent]
+    public sealed class ChargerActiveComponent : Component
```

Comment:
Most of these types of components are named "**Active**SomethingComponent" rather than "Something**Active**Component"